packages/liveblocks-react-ui/src/primitives/Comment/index.tsx

```diff
@@ -111,7 +111,7 @@ const CommentBody = forwardRef<HTMLDivElement, CommentBodyProps>(
 
                       return (
                         <Link href={href} key={index}>
-                          {inline.url}
+                          {inline.text ?? inline.url}
```

Comment:
Can `inline.text` ever be `""`? If so, maybe in this case, we should use `||` instead of `??` (which I'd normally always prefer).

Reply:
No that would mean a 0-characters link and Slate's normalization process prevents empty nodes from existing (they either get merged with compatible neighbors or get removed).